packages/zowe-explorer/src/Profiles.ts

```diff
@@ -401,20 +400,7 @@ export class Profiles extends ProfilesCache {
                     }
                     break;
                 case "port":
-                    if (updSchemaValues[value] === undefined) {
-                        updPort = await this.portInfo(value, schema);
-                        if (Number.isNaN(updPort)) {
-                            vscode.window.showInformationMessage(
-                                localize(
-                                    "editConnection.undefined.port",
-                                    "Invalid Port number provided or operation was cancelled"
-                                )
-                            );
-                            return undefined;
-                        }
-                        updSchemaValues[value] = updPort;
-                        break;
-                    }
+                    // Do nothing, the user should put in the port at the end of the hostname
```

Comment:
please bring the prompt for the port back. The user should have the ability to put the port if they didn't put it in the host

Reply:
I disagree with this. Showing the user that they can include the port in the URL, and then asking them for the port specifically, amounts to asking them twice for the port. Putting the port at the end of the URL seems like basic knowledge to me, everyone will understand how to do this, it's not like we are being unclear somehow about how to specify your port.

I would prefer to ask the team before putting it back the way it was.

Reply:
Maybe I can propose to add some validation of user value as a compromise? 

Like to force user to include port in the first dialog and disallow them to continue profile creation without it? Like a runtime validation?

Reply:
What if they don't want to add the port? It seems weird to force them to put in a zero, we should be able to tell this if they simply don't add anything. And if they decide later they want to add the port they can just go through the edit dialog again.

Reply:
Hmm, but they have to specify it, right?

And we have this updated validation function now, that can handle default port values, so even if they will not specify anything, but protocol will be `https` - everything will be fine.

Reply:
@katelynienaber let's talk offline

Reply:
With the updated validation function, they don't have to specify the port, we will use 443 as default if it isn't specified.

So, since they don't have to specify it, if they don't we should just deal with it, not give them some validation error imho

Reply:
Why would you force the user to enter the port or the protocol in the hostname textbox?  We changed this before to allow the user to have both options.  If it's entered as a whole URL with protocol, hostname, and port this entry should be split up into appropriate pieces and if a piece was missing a following prompt would show up to get the missing information.  The point of this issue is to not re-prompt if it was already entered.

Reply:
I just tested PR and only entered hostname and was never prompted for port while creating an RSE profile and it automatically assigned 443 which would be incorrect
![image](https://user-images.githubusercontent.com/49491949/117977923-4e009e00-b2ff-11eb-82ea-b802dedbb938.png)

Reply:
@nickImbirev @jellypuno @JillieBeanSim Please check this again. I wasn't aware it had been discussed previously.

When creating a new profile or editing an old one, it should now...

Prompt you for port specifically if you enter after the URL:
- Nothing
- 0
- Something that is NaN

It will not prompt you for port if you enter:
- 443
- Any other number